### PR TITLE
commit 11111

### PR DESCRIPTION
**Pull request by kocho1984**

### CODE REVIEW

File: file001.txt [hunk 1]
```diff
@@ -7,6 +7,10 @@ Yay! And now I'm adding a new line of text using vcode! Finally!
 
 I'm a happy camper!
 
+mako 11111111111
```
Comment (Geek-Cat666) [inline file001.txt]: Dupa 1!

File: file001.txt [hunk 2]
```diff
@@ -7,6 +7,10 @@ Yay! And now I'm adding a new line of text using vcode! Finally!
...
+
+mako 222222222
```
Comment (Geek-Cat666) [inline file001.txt]: Coment2

File: file001.txt (hunk 1 above)
Comment (Geek-Cat666) [reply file001.txt]: Coment 2 w rev1

**Review by Geek-Cat666 — CHANGES_REQUESTED** [review]
Summary comment!

File: file001.txt (hunk 1 above)
Comment (kocho1984) [reply file001.txt]: Done.

File: file001.txt (hunk 2 above)
Comment (kocho1984) [reply file001.txt]: Done.

**Comment by Geek-Cat666** [comment]
> Done. 

OK!

**Review by Geek-Cat666 — APPROVED** [review]

**Pull request merged**